### ISSUE TITLE
Lastest post do not process tv's

### ISSUE THREAD

**Issue by eighthday** [modxcms/Articles#19]

**Comment by jpdevries** [comment]
How would one reproduce this?

**Comment by gumby-man** [comment]
This is mentioned as issue in MODX forums as well.

http://forums.modx.com/thread/78005/tv-in-articleslatestposttpl#dis-post-499531

It is in "core\components\articles\model\articles\articlescontainer.class.php", that getLatestPostsCall() does not use includeTVs etc. in getResources. I patched it adding four lines:

```
        &includeTVs=`'.$this->xpdo->getOption('archivesIncludeTVs',$settings,0).'`
        &includeTVsList=`'.$this->xpdo->getOption('includeTVsList',$settings,'').'`
        &processTVs=`'.$this->xpdo->getOption('archivesProcessTVs',$settings,0).'`
        &processTVsList=`'.$this->xpdo->getOption('processTVsList',$settings,'').'`
```

For me this is a bug, I don't see any reason why TVs are not available here (while they are in ArticleRowTpl).

**Comment by gumby-man** [comment]
Seems to be fixed in 1.7.10, but I cannot find the corresponding issue here.

**Comment by JoshuaLuckers** [comment]
This issue is fixed therefore I'm closing this issue.
